tests/haskell_proto_library/BUILD.bazel

```diff
@@ -76,6 +76,7 @@ haskell_library(
         ":person_haskell_proto",
         ":stripped_address_haskell_proto",
         "//tests/hackage:base",
+        "@stackage//:proto-lens",
```

Comment:
```suggestion
```
I'm not sure this is still needed, I think this was an artifact of updating the proto toolchain, which is no longer necessary.